pkg/util/utils.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2022 Antrea Authors.
```

Comment:
I also tried to move some utility functions to the common pkg in PR #138 . I think most of the functions are similar, but I would like to get some feedbacks about if we prefer to put all these functions in one file like this one, or classify them into multiple pkgs like clickhouse, policyrecommendation, k8s as I do in PR #138

Reply:
I don't have strong opinion regarding this, but I might prefer classifying them into multiple pkgs.